eng/tools/generator/cmd/v2/common/cmdProcessor.go

```diff
@@ -16,11 +19,54 @@ import (
 func ExecuteGoGenerate(path string) error {
 	cmd := exec.Command("go", "generate")
 	cmd.Dir = path
-	output, err := cmd.CombinedOutput()
-	log.Printf("Result of `go generate` execution: \n%s", string(output))
+
+	stdoutPipe, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+
+	stderrPipe, err := cmd.StderrPipe()
 	if err != nil {
-		return fmt.Errorf("failed to execute `go generate` '%s': %+v", string(output), err)
+		return err
+	}
+
+	if err = cmd.Start(); err != nil {
+		return err
+	}
+
+	var stdoutBuffer bytes.Buffer
+	if _, err = io.Copy(&stdoutBuffer, stdoutPipe); err != nil {
+		return err
+	}
+
+	var stderrBuffer bytes.Buffer
+	if _, err = io.Copy(&stderrBuffer, stderrPipe); err != nil {
+		return err
+	}
+
+	err = cmd.Wait()
+
+	fmt.Println(stdoutBuffer.String())
+	if stdoutBuffer.Len() > 0 {
+		if strings.Contains(stdoutBuffer.String(), "error   |") {
+			// find first error message until last
+			errMsgs := stdoutBuffer.Bytes()
+			index := regexp.MustCompile(`error   \|`).FindIndex(errMsgs)
+			if len(index) == 2 {
+				return fmt.Errorf("failed to execute `go generate`:\n%s", string(errMsgs[index[0]:]))
```

Comment:
Let's add the next steps:
Refer to the detail errors for potential fixes.
If the issue persists after re-running, contact the Go language support channel at https://aka.ms/azsdk/go-lang-teams-channel and include this spec pull request.